komodo/data/build__python-virtualenv.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -xe
+
+while test $# -gt 0; do
+    case "$1" in
+        --fakeroot)
+            shift
+            export FAKEROOT=$1
+            ;;
+       --prefix)
+            shift
+            export PREFIX=$1
+            ;;
+        --virtualenv)
+            shift
+            export VIRTUALENV=$1
+            ;;
+        --virtualenv-interpreter)
+            shift
+            export VIRTUALENV_INTEPRETER=$1
+            ;;
+        *)
+            export OPTS="$OPTS $1"
+            ;;
+    esac
+    shift
+done
+
+ $VIRTUALENV -p $VIRTUALENV_INTEPRETER -v --always-copy --never-download --no-setuptools --no-pip --no-wheel ${FAKEROOT}${PREFIX} 1>&2
```

Comment:
${FAKEROOT}${PREFIX} should result in a path? what if missing /?

Reply:
Will fix.